bokeh/application/handlers/code_runner.py

```diff
@@ -217,6 +211,16 @@ def run(self, module, post_check=None):
 # Private API
 #-----------------------------------------------------------------------------
 
+def _handle_exception(handler, e):
+    """helper to stash an exception on a CodeRunner or Handler"""
+    handler._failed = True
+    handler._error_detail = traceback.format_exc()
+
+    _exc_type, _exc_value, exc_traceback = sys.exc_info()
+    filename, line_number, func, txt = traceback.extract_tb(exc_traceback)[-1]
+
+    handler._error = "%s\nFile \"%s\", line %d, in %s:\n%s" % (str(e), os.path.basename(filename), line_number, func, txt)
+
```

Comment:
This seems pretty independent of `CodeRunner`. Perhaps this would be better in `handler.py` ? (with the doctring updated)

Reply:
sounds fine